src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -340,4 +340,13 @@ describe('ReactIncrementalScheduling', () => {
     // animation priority.
     expect(ReactNoop.getChildren()).toEqual([span(1)]);
   });
+
+  it('can force synchronous updates with syncUpdates, even inside batchedUpdates', () => {
+    ReactNoop.batchedUpdates(() => {
+      ReactNoop.syncUpdates(() => {
+        ReactNoop.render(<span />);
+        expect(ReactNoop.getChildren()).toEqual([span()]);
```

Comment:
Not sure what you mean. All `batchedUpdates` does it make it so any sync updates are batched together using Task priority. `syncUpdates` does the opposite, where if you're already in a batch, it forces a synchronous flush.

The test only passes if line 347 flushes synchronously. So I'm not sure how you'd get a false positive.